aml-development/frontend
# fix(notifications): update mark/unmark all buttons

AMLOS-812

**Description**
Click on the Notifications bell icon. A few notifications are visible, however you want to see the full list. Click on the See more.

On the notifications screen,

Since eye icons are used for edit a listing for a different function, we'll remove the eyes and just use "Mark All as Read", and remove Mark Unread.

**Acceptance Criteria**
Notifications page has a button labeled "Mark All as Read" as the only option in the header

**How to Test**
Pull this branch
Go to notifications page (ie click _See More_ when opening your notifications)
Verify that header only has a button labeled "Mark All as Read"
Verify that clicking the button marks all notifications read
